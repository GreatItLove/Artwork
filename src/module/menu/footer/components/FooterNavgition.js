import React from 'react';
import { BottomNavigation, BottomNavigationItem } from 'material-ui/BottomNavigation';
import FontIcon from 'material-ui/FontIcon';

/**
 * @function FooterNavgition
 * @returns {XML}
 */
function FooterNavgition() {
  return (
    <BottomNavigation style={{ backgroundColor: 'none' }}>
      <BottomNavigationItem label="Support" icon={<FontIcon className="material-icons">phone</FontIcon>} />
      <BottomNavigationItem label="Tutorials" icon={<FontIcon className="material-icons">library_books</FontIcon>} />
      <BottomNavigationItem
        label="Terms of Service"
        icon={<FontIcon className="material-icons">view_headline</FontIcon>}
      />
    </BottomNavigation>
  );
}

export default FooterNavgition;
