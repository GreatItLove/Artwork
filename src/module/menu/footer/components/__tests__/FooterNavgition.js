import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import FooterNavgition from '../FooterNavgition';

describe('FooterNavgition', () => {
  it('Snapshot of FooterNavgition', () => {
    const wrapper = shallow(<FooterNavgition />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
