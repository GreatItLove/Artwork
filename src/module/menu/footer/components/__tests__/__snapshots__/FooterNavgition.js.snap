// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterNavgition Snapshot of FooterNavgition 1`] = `
<BottomNavigation
  style={
    Object {
      "backgroundColor": "none",
    }
  }
>
  <BottomNavigationItem
    icon={
      <FontIcon
        className="material-icons"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        phone
      </FontIcon>
    }
    label="Support"
  />
  <BottomNavigationItem
    icon={
      <FontIcon
        className="material-icons"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        library_books
      </FontIcon>
    }
    label="Tutorials"
  />
  <BottomNavigationItem
    icon={
      <FontIcon
        className="material-icons"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
      >
        view_headline
      </FontIcon>
    }
    label="Terms of Service"
  />
</BottomNavigation>
`;
